src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -24,7 +24,9 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        Driver driver = new Driver(name, licenseNumber, login, password);
         driverService.create(driver);
         resp.sendRedirect(req.getContextPath() + "/drivers/add");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/drivers");
```
better change to drivers , so after adding you can see if driver is added or not ? up to you^_^

Reply:
Thanks, also thought about it. but decided to leave like that in order to add several drivers at once for testing my app, the same with other objects :)